src/components/Requirements/IncompleteSelfCheck.vue

```diff
@@ -104,10 +158,25 @@ export default Vue.extend({
 <style scoped lang="scss">
 @import '@/assets/scss/_variables.scss';
 
+.separator {
+  height: 1px;
+  width: 100%;
+  background-color: $inactiveGray;
+  margin-top: 1rem;
+}
+
+.top {
+  font-size: 14px;
+  line-height: 17px;
+  color: #757575;
```

Comment:
Nit: I think we have an scss variable

Reply:
You're right we do my bad, but it looks like this hex code is used in a few different places. Making a notion item for myself to fix this soon!